### DISCUSSION TITLE
Add an alternative user-friendly FPS display

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#14437] [Ideas] [answered]
There is how it was on previous releases (2021.523.0): 
![ScreenShot_20210821171804](https://user-images.githubusercontent.com/86934170/130324737-e90fb1fc-e05c-4a77-8a33-c64783ba691d.png)
And how it looks like right now: 
![ScreenShot_20210821171457](https://user-images.githubusercontent.com/86934170/130324761-8a87d4af-9ecc-4352-884a-35781a061c51.png)
Please, fix the font of the counter.

**Comment by frenzibyte** [comment]
The FPS overlay comes from the framework which has its own font (Roboto) instead, the fact that it was using the game's font (Torus) on 2021.523.0 and older is because the framework was incorrectly considering the first loaded font for such overlays, which now has been corrected to use a specific font for all debugging overlays.

I suppose it should still match the game's font given that it's more of a user-readable statistic overlay rather than a debug overlay.

**Reply by Loreos7** [reply]
I use it to control the delay in ms. That's why i think so for the Torus.

**Comment by peppy** [comment]
The framework fps display is not intended to be something we show to the average user. It will eventually be replaced.

**Reply by Loreos7** [reply]
Then I'd like to have a customizable fps counter for each section (input, audio, draw) and place on screen. I'll wait for it in new versions of osu

**Reply by peppy** [reply]
If anything we would hide and simplify all but update. Most are not relevant. Best to wait and see anyways.

**Comment by Loreos7** [comment, marked as answer]
https://github.com/ppy/osu/pull/19250
